TeslaMateAgile/Services/EnerginetService.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Microsoft.Extensions.Options;
+using System;
+using System.Diagnostics;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using TeslaMateAgile.Data;
+using TeslaMateAgile.Data.Options;
+using TeslaMateAgile.Services.Interfaces;
+
+namespace TeslaMateAgile.Services;
+
+public class EnerginetService : IPriceDataService
+{
+    private readonly HttpClient _client;
+    private readonly EnerginetOptions _options;
+    private readonly FixedPriceOptions _fixedPricesOptions;
+    private readonly FixedPriceService _fixedPriceService;
+
+    public EnerginetService(HttpClient client, IOptions<EnerginetOptions> options)
+    {
+        _client = client;
+        _options = options.Value;
+
+        _fixedPricesOptions = _options.FixedPrices;
+        _fixedPriceService = new FixedPriceService(Options.Create(_fixedPricesOptions));
+    }
+
+    public async Task<IEnumerable<Price>> GetPriceData(DateTimeOffset from, DateTimeOffset to)
+    {
+        var url = "Elspotprices?offset=0&start=" + from.AddHours(-2).UtcDateTime.ToString("yyyy-MM-ddTHH:mm") + "&end=" + to.AddHours(2).UtcDateTime.ToString("yyyy-MM-ddTHH:mm") + "&filter={\"PriceArea\":[\"" + _options.Region + "\"]}&sort=HourUTC ASC&timezone=dk".Replace(@"\", string.Empty); ;
```

Comment:
Should the timezone parameter here be `dk` presumably for Denmark? Is there a way to set it to UTC instead perhaps? This may explain your need to -2 and +2 hours

Reply:
Currently the timezone is unfortunately not a parameter in their query